zips/zip-0230.rst

```diff
@@ -315,31 +321,39 @@ We introduce the ``nAGExpiryHeight`` field in this transaction format in order t
 For the OrchardZSA protocol, which does not make use of an additional expiry height for transactions, we set the value of ``nAGExpiryHeight`` to be ``0`` by consensus.
```

Comment:
We need to ensure that a conformance requirement exists within the specification that requires `nAGExpiryHeight` to be zero as of NU7.